### DISCUSSION TITLE
Alternative ways to handle undefined

### DISCUSSION THREAD

**Discussion by flappyBug** [barvian/number-flow#81] [Ideas]
`<NumberFlow value={undefined} />` renders as `NaN`. It better renders as empty.

**Comment by barvian** [comment]
Hi, thanks for filing. I don't have a strong feeling about this, but I like being consistent with `Intl.NumberFormat` which currently returns NaN for undefined. I'll convert this to a discussion though, in case other people want to chime in 👋

**Reply by flappyBug** [reply]
@barvian Perhaps rendering `undefined` as `NaN` is acceptable, as I can control NumberFlow through conditional rendering. However, in my use case, I want to display an animation transitioning from empty to a number, and it seems that NumberFlow does not support an animation on show option.

**Reply by barvian** [reply]
Hi @flappyBug, what type of entrance/exit animations are you trying to build?
